AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -221,7 +221,8 @@
       "properties": {
         "description": "[concat(variables('nodeRecordingRuleGroupDescription'), variables('version'))]",
         "scopes": [
-          "[parameters('azureMonitorWorkspaceResourceId')]"
+          "[parameters('azureMonitorWorkspaceResourceId')]",
```

Comment:
What benefit does adding the cluster resource id to the scope give us? Does that mean we do not need to specify the clusterName parameter?

Reply:
Based on the PR description, would these alerts show up in the Alerts Rules page on the AKS cluster if we set this scope?

Reply:
Thanks Kaveesh! Yes it would show under the cluster in the rules UI which the rules team is about to launch.